src/rust/engine/process_execution/src/remote.rs

```diff
@@ -312,7 +312,7 @@ impl CommandRunner {
     platform_properties: BTreeMap<String, String>,
     thread_count: usize,
     store: Store,
-    futures_timer_thread: resettable::Resettable<futures_timer::HelperThread>,
+    futures_timer_thread: Arc<futures_timer::HelperThread>,
```

Comment:
Unrelated, but: now that we guarantee the tokio runtime everywhere, we should definitely switch to using tokio's timers.

Reply:
https://github.com/pantsbuild/pants/pull/7822